ingestion/src/metadata/utils/helpers.py

```diff
@@ -280,3 +280,16 @@ def find_column_in_table(column_name: str, table: Table) -> Optional[Column]:
     return next(
         (col for col in table.columns if col.name.__root__ == column_name), None
     )
+
+
+def list_to_dict(original: Optional[List[str]], sep: str = "=") -> Dict[str, str]:
+    """
+    Given a list with strings that have a separator,
+    convert that to a dictionary of key-value pairs
+    """
+    if not original:
+        return {}
+
+    split_original = [(elem.split(sep)) for elem in original if sep in elem]
+
+    return {elem[0]: elem[1] for elem in split_original}
```

Comment:
maybe here we can eliminate using loops twice 🤔 

```
split_original = [(elem.split(sep)[:2]) for elem in original if sep in elem]
return dict(split_original)
```

this should work, wdyt 🤔

Reply:
good call, I'll change it. THanks!

Reply:
pushed with a slight change so that we help enforce the right typing